styles/lichess/catppuccin.user.css

```diff
@@ -0,0 +1,567 @@
+/* ==UserStyle==
+@name lichess Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/lichess
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/lichess
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/lichess/catppuccin.user.css
+@description Soothing pastel theme for lichess
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select flavor "Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
```

Comment:
Looks like there is a light/dark/auto theme toggle, `body[data-theme="x"]`. Possible values are "light", "dark", and "system" - you could do something like:

```css
	body[data-theme="light"] {
		:root {
			#catppuccin(@lightFlavor, @accentColor);
		}
	}
	body[data-theme="dark"] {
		:root {
			#catppuccin(@darkFlavor, @accentColor);
		}
	}
	body[data-theme="system"] {
		@media (prefers-color-scheme: light) {
			:root {
				#catppuccin(@lightFlavor, @accentColor);
			}
		}
		@media (prefers-color-scheme: dark) {
			:root {
				#catppuccin(@darkFlavor, @accentColor);
			}
		}
	}